src/Nethermind/Nethermind.JsonRpc/Modules/Admin/PeerInfo.cs

```diff
@@ -39,18 +41,20 @@ public PeerInfo(Peer peer, bool includeDetails)
 
             Name = peer.Node.ClientId;
             Id = peer.Node.Id.Hash.ToString(false);
-            Host = peer.Node.Host is null ? null : IPAddress.Parse(peer.Node.Host).MapToIPv4().ToString();
+            Host = IPAddress.Parse(peer.Node.Host!).MapToIPv4().ToString();
```

Comment:
Are we sure it will always be not null?

Reply:
assumed every node would/should have a host

Reply:
@MarekM25 
how do you know when a node(peer) is trusted?